source/patterns/@aws-solutions-constructs/core/lib/ssm-parameter-helper.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ *  Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import {StringParameter, StringParameterProps} from '@aws-cdk/aws-ssm';
+import {Construct} from '@aws-cdk/core';
+
+/**
+ * Method to build the default AWS SSM Parameter Store
+ *
+ * @param scope
+ * @param id
+ * @param stringParameterProps
+ */
+export function buildSsmStringParamter(scope: Construct, id: string, stringParameterProps: StringParameterProps): StringParameter {
+  return new StringParameter(scope, id, stringParameterProps);
```

Comment:
So I'm looking at possible defaults, and see that there really aren't any. But I also see parameter Type as a member of StringParameterProps. What happens if you send SECURE_STRING or STRINGLIST? Does it get ignored? The docs are pretty clear that you can't create a SECURE_STRING parameter with the CDK, and StringList has a completely separate  L2 construct.

If it must be STRING, then we should overrule whatever the user sent and dump a warning that we are doing so. Also should point it out in the docs.